### ISSUE TITLE
Consider height of the container as well

### ISSUE THREAD

**Issue by MarcMagnin** [malte-wessel/react-textfit#29]
Again thanks a lot for that library, it works like a charm.
But again I'm coming with more question, is it possible to consider the height of the container as well?

I have a component that is fully resizable and that can end up in such situation:
![image](https://user-images.githubusercontent.com/725667/28929705-fba985a8-7870-11e7-9447-30344be62fac.png)

Many thanks!

**Comment by denis-sokolov** [comment]
I’m pretty sure we’re supposed to handle the height of the container as well. It’s either a weird setup you have or a bug in our library. Please try to isolate your behavior and show a demo of the breakage on a website like jsbin.com, then reopen the issue for us to investigate.

**Comment by MarcMagnin** [comment]
Cheers, tricky to setup a jsbin but I'll do later if I find the time.
Definitly it doesn't consider the height even if I set the height as static such as 40px.
I've tried as well to modify the css on the demo page you provide:
![image](https://user-images.githubusercontent.com/725667/28963940-2890fb88-790b-11e7-9b73-4daa1757ec5d.png)

But maybe that's related to the library resize function not getting called.
